cfme/tests/test_appliance.py

```diff
@@ -152,9 +152,8 @@ def test_memory_total(appliance):
         testtype: functional
         casecomponent: Appliance
     """
-    result = appliance.ssh_client.run_command(
-        r'free -g | grep Mem: | awk \'{ print $2 }\'').output
-    assert int(result) >= 6
+    result = appliance.ssh_client.run_command(r"free -g | grep Mem: | awk '{ print $2 }'").output
+    assert int(result.strip()) >= 6
```

Comment:
I did the same thing in a branch, but the strip is unnecessary here:
```
In [7]: int(app.ssh_client.run_command(r"free -g | grep Mem: | awk '{ print $2 }'").output)
Running command "free -g | grep Mem: | awk '{ print $2 }'"
Out[7]: 11
```

Reply:
Also, this fix is in 8672 already:
https://github.com/ManageIQ/integration_tests/pull/8672/files#diff-07e1dc45e2c0264896d4bceb289d1fa0L155

But I'm not opposed to merging separately.

Reply:
Now that I say that, using single quotes still causes issues in 8672, I'll update.

Reply:
I think srip is better before conversion :). @mshriver if you are going to merge #8672 then good to close this. I just found this while reviewing one of PR.

Reply:
Cool. I'd disagree about the strip, because its unnecessary, the `int()` call handles it.  Similar to how we don't use `dict.get(key, None)` because `None` is the default implicit behavior, here its an extra call to strip that's implicitly handled by the int() conversion.